{"time":"2025-04-14T00:49:47.4206937+05:30","level":"INFO","msg":"stream: starting","core version":"0.19.8","symlink path":"c:\\Users\\prajv\\DA6401_ASSIGNMENT2\\wandb\\run-20250414_004947-e4h83q5z\\logs\\debug-core.log"}
{"time":"2025-04-14T00:49:48.7538083+05:30","level":"INFO","msg":"created new stream","id":"e4h83q5z"}
{"time":"2025-04-14T00:49:48.7538083+05:30","level":"INFO","msg":"stream: started","id":"e4h83q5z"}
{"time":"2025-04-14T00:49:48.7538083+05:30","level":"INFO","msg":"handler: started","stream_id":"e4h83q5z"}
{"time":"2025-04-14T00:49:48.7538083+05:30","level":"INFO","msg":"writer: Do: started","stream_id":"e4h83q5z"}
{"time":"2025-04-14T00:49:48.7538083+05:30","level":"INFO","msg":"sender: started","stream_id":"e4h83q5z"}
{"time":"2025-04-14T00:49:49.1963544+05:30","level":"INFO","msg":"Starting system monitor"}
{"time":"2025-04-14T01:07:42.3503141+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/da6401-assignments/assignment2/e4h83q5z/file_stream\": dial tcp 35.186.228.49:443: connectex: A socket operation was attempted to an unreachable host."}
